draft-protected-headers.in.md

```diff
@@ -200,122 +254,120 @@ It is NOT RECOMMENDED to generate messages with such complicated structures.
 Even if a receiving MUA can parse this structure properly, it is nearly impossible to render in a way that the user can reason about the cryptographic properties of part O compared to part Q.
 
 
-Original Headers are Outside
-----------------------------
+Exposed Headers are Outside
+---------------------------
 
 The Cryptographic Envelope fully encloses the Cryptographic Payload, whether the message is signed or encrypted or both.
-The Original Headers are considered to be outside of both.
+The Exposed Headers are considered to be outside of both.
 
 
 Message Composition
 ===================
 
-The Protected Headers scheme is summarized as follows:
+This section describes the composition of a cryptographically-protected message with Protected Headers.
 
-   1. All message headers known to the MUA at composition time MUST be copied verbatim into the first MIME header of the Cryptographic Payload.
-   2. When encrypting, Exposed Headers MAY be *obscured* by a transformation (including deletion).
-   3. When encrypting, if the MUA has obscured any user-facing header data, it SHOULD add a Legacy Display Part to the Cryptographic Payload which duplicates this information.
+We document legacy composition of cryptographically-protected messages (without protected headers) in {{legacy-composition}}, and then describe a revised version of that algorithm in {{protected-header-composition}} that produces conformant Protected Headers.
 
-*Note:* The above is not a description of a sequential algorithm.
+Copying All Headers
+-------------------
 
-Details:
+All non-structural headers known to the composing MUA are copied to the MIME header of the Cryptographic Payload.
```

Comment:
Should all defined terms appear in ```` or with 1st letter capitalized?, here:
s/All non-strucutural headers/All non-`Structural Headers`/

Reply:
I've been trying to use `code` style to indicate literal textual elements, and Upper Case to indicate defined terms.